spot-setup/hdfs_setup.sh

```diff
@@ -7,8 +7,8 @@ source /etc/spot.conf
 #
 # creating HDFS user's folder
 #
-sudo -u hdfs hadoop fs -mkdir ${HUSER}
-sudo -u hdfs hadoop fs -chown ${USER}:supergroup ${HUSER}
+hadoop fs -mkdir ${HUSER}
```

Comment:
if the user is part of superuser group, they should have permissions to add folders.
we could do some additional testing there.

the problem that this resolves would be that if you have a kerberos ticket that has been assigned to a user, for sake of argument the username is Spot. when running sudo -u you become the hdfs user, and must have a ticket for that user.

we can do some additional testing, but i'm in support of this change.